Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -0,0 +1,603 @@
+            paymentMethodTypes = self.paymentMethodTypes
+        }
+        let confirmHandler: PaymentSheet.IntentConfiguration.ConfirmHandler = { [weak self] in
+            self?.confirmHandler($0, $1, $2)
+        }
+        switch settings.mode {
+        case .payment:
+            return PaymentSheet.IntentConfiguration(
+                mode: .payment(amount: amount!, currency: settings.currency.rawValue, setupFutureUsage: nil),
+                paymentMethodTypes: paymentMethodTypes,
+                confirmHandler: confirmHandler
+            )
+        case .paymentWithSetup:
+            return PaymentSheet.IntentConfiguration(
+                mode: .payment(amount: amount!, currency: settings.currency.rawValue, setupFutureUsage: .offSession),
+                paymentMethodTypes: paymentMethodTypes,
+                confirmHandler: confirmHandler
+            )
+        case .setup:
+            return PaymentSheet.IntentConfiguration(
+                mode: .setup(currency: settings.currency.rawValue, setupFutureUsage: .offSession),
+                paymentMethodTypes: paymentMethodTypes,
+                confirmHandler: confirmHandler
+            )
+        }
+    }
+
+    var clientSecret: String?
+    var ephemeralKey: String?
+    var customerID: String?
+    var paymentMethodTypes: [String]?
+    var amount: Int?
+    var checkoutEndpoint: String = PaymentSheetTestPlaygroundSettings.defaultCheckoutEndpoint
+    var addressViewController: AddressViewController?
+    var appearance = PaymentSheet.Appearance.default
+
+    func makeAlertController() -> UIAlertController {
+        let alertController = UIAlertController(
+            title: "Complete", message: "Completed", preferredStyle: .alert)
+        let OKAction = UIAlertAction(title: "OK", style: .default) { (_) in
+            alertController.dismiss(animated: true, completion: nil)
+        }
+        alertController.addAction(OKAction)
+        return alertController
+    }
+
+    private var subscribers: Set<AnyCancellable> = []
+
+    init(settings: PaymentSheetTestPlaygroundSettings) {
+        // Enable experimental payment methods.
+        //        PaymentSheet.supportedPaymentMethods += [.link]
+        PaymentSheet.enableACHV2InDeferredFlow = true // TODO(https://jira.corp.stripe.com/browse/BANKCON-6731) Remove this.
+
+        // Hack to ensure we don't force the native flow unless we're in a UI test
+        if ProcessInfo.processInfo.environment["UITesting"] == nil {
+            UserDefaults.standard.removeObject(forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        } else {
+            // This makes the Financial Connections SDK use the native UI instead of webview. Native is much easier to test.
+            UserDefaults.standard.set(true, forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE")
+        }
+        self.settings = settings
+
+        $settings.sink { _ in
+            self.load()
```

Comment:
One reason to not have this auto reload behavior is that i often fiddle around with settings before i want things to reload, because i have a ton of breakpoints set up, or i'm hitting localhost w/ a bunch of print statements and it can be noisy. I think I prefer having an explicit load function -- or maybe an option which allows me to turn off autoload?

Reply:
Mind if I add that in a follow-up PR in the next few business days? I have a plan for implementing that correctly (with a nice UI to show the "dirty" state if the current PaymentSheet's settings don't match), but I haven't gotten around to it yet.